pkg/skaffold/cluster/minikube.go

```diff
@@ -162,21 +130,28 @@ func matchServerURLFor(kubeContext string, serverURL *url.URL) (bool, error) {
 		return false, fmt.Errorf("failed to unmarshal data: %w", err)
 	}
 
+	vmTypeDrivers := map[string]bool{
```

Comment:
This needs a comment.  I wonder if it's better to invert this condition to identify non-VM drivers?

Reply:
Removed this too.